src/runtime/node/crypto/internal/constants.ts

```diff
@@ -0,0 +1,221 @@
+    SSL_OP_NO_RENEGOTIATION: 1_073_741_824,
+    SSL_OP_NO_SESSION_RESUMPTION_ON_RENEGOTIATION: 65_536,
+    SSL_OP_NO_SSLv2: 0,
+    SSL_OP_NO_SSLv3: 33_554_432,
+    SSL_OP_NO_TICKET: 16_384,
+    SSL_OP_NO_TLSv1: 67_108_864,
+    SSL_OP_NO_TLSv1_1: 268_435_456,
+    SSL_OP_NO_TLSv1_2: 134_217_728,
+    SSL_OP_NO_TLSv1_3: 536_870_912,
+    SSL_OP_PRIORITIZE_CHACHA: 2_097_152,
+    SSL_OP_TLS_ROLLBACK_BUG: 8_388_608,
+    ENGINE_METHOD_RSA: 1,
+    ENGINE_METHOD_DSA: 2,
+    ENGINE_METHOD_DH: 4,
+    ENGINE_METHOD_RAND: 8,
+    ENGINE_METHOD_EC: 2048,
+    ENGINE_METHOD_CIPHERS: 64,
+    ENGINE_METHOD_DIGESTS: 128,
+    ENGINE_METHOD_PKEY_METHS: 512,
+    ENGINE_METHOD_PKEY_ASN1_METHS: 1024,
+    ENGINE_METHOD_ALL: 65_535,
+    ENGINE_METHOD_NONE: 0,
+    DH_CHECK_P_NOT_SAFE_PRIME: 2,
+    DH_CHECK_P_NOT_PRIME: 1,
+    DH_UNABLE_TO_CHECK_GENERATOR: 4,
+    DH_NOT_SUITABLE_GENERATOR: 8,
+    RSA_PKCS1_PADDING: 1,
+    RSA_NO_PADDING: 3,
+    RSA_PKCS1_OAEP_PADDING: 4,
+    RSA_X931_PADDING: 5,
+    RSA_PKCS1_PSS_PADDING: 6,
+    RSA_PSS_SALTLEN_DIGEST: -1,
+    RSA_PSS_SALTLEN_MAX_SIGN: -2,
+    RSA_PSS_SALTLEN_AUTO: -2,
+    defaultCoreCipherList:
+      "TLS_AES_256_GCM_SHA384:TLS_CHACHA20_POLY1305_SHA256:TLS_AES_128_GCM_SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-AES256-GCM-SHA384:DHE-RSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-SHA256:DHE-RSA-AES128-SHA256:ECDHE-RSA-AES256-SHA384:DHE-RSA-AES256-SHA384:ECDHE-RSA-AES256-SHA256:DHE-RSA-AES256-SHA256:HIGH:!aNULL:!eNULL:!EXPORT:!DES:!RC4:!MD5:!PSK:!SRP:!CAMELLIA",
+    TLS1_VERSION: 769,
+    TLS1_1_VERSION: 770,
+    TLS1_2_VERSION: 771,
+    TLS1_3_VERSION: 772,
+    POINT_CONVERSION_COMPRESSED: 2,
+    POINT_CONVERSION_UNCOMPRESSED: 4,
+    POINT_CONVERSION_HYBRID: 6,
+    get defaultCipherList() {
+      return "TLS_AES_256_GCM_SHA384:TLS_CHACHA20_POLY1305_SHA256:TLS_AES_128_GCM_SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-AES256-GCM-SHA384:DHE-RSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-SHA256:DHE-RSA-AES128-SHA256:ECDHE-RSA-AES256-SHA384:DHE-RSA-AES256-SHA384:ECDHE-RSA-AES256-SHA256:DHE-RSA-AES256-SHA256:HIGH:!aNULL:!eNULL:!EXPORT:!DES:!RC4:!MD5:!PSK:!SRP:!CAMELLIA";
+    },
+    set defaultCipherList(_ignored) {},
+  },
+
+  // collected from Node.js v22.3.0 using:
+  // require('crypto').constants
+  // require('crypto').constants.defaultCipherList
+  ...{
+    OPENSSL_VERSION_NUMBER: 805_306_576,
+    SSL_OP_ALL: 2_147_485_776,
+    SSL_OP_ALLOW_NO_DHE_KEX: 1024,
+    SSL_OP_ALLOW_UNSAFE_LEGACY_RENEGOTIATION: 262_144,
+    SSL_OP_CIPHER_SERVER_PREFERENCE: 4_194_304,
+    SSL_OP_CISCO_ANYCONNECT: 32_768,
+    SSL_OP_COOKIE_EXCHANGE: 8192,
+    SSL_OP_CRYPTOPRO_TLSEXT_BUG: 2_147_483_648,
+    SSL_OP_DONT_INSERT_EMPTY_FRAGMENTS: 2048,
+    SSL_OP_LEGACY_SERVER_CONNECT: 4,
+    SSL_OP_NO_COMPRESSION: 131_072,
+    SSL_OP_NO_ENCRYPT_THEN_MAC: 524_288,
+    SSL_OP_NO_QUERY_MTU: 4096,
```

Comment:
Most of these are repeated 4 times i think we san safely merge them to reduce size and still fulfill purpose of polyfill

Reply:
there are some differences between node versions, but you are right, payload size matters. I'll dedupe.